plugins/generator-addon-1.19.x/addon-1.19.x/mappings/blocksitems.yaml

```diff
@@ -875,6 +875,7 @@ Blocks.MANGROVE_WOOD: "mangrove_wood"
 Blocks.STRIPPED_MANGROVE_WOOD: "stripped_mangrove_wood"
 Blocks.MANGROVE_LEAVES: "mangrove_leaves"
 Blocks.MANGROVE_STAIRS: "mangrove_stairs"
+Blocks.MANGROVE_SIGN: "mangrove_standing_sign"
 Blocks.MANGROVE_WALL_SIGN: "mangrove_wall_sign"
 Blocks.MANGROVE_DOOR: "mangrove_door"
 Blocks.MANGROVE_TRAPDOOR: "mangrove_trapdoor"
```

Comment:
`Blocks.POTTED_MANGROVE_PROPAGULE` (a few lines lower) somehow doesn't have a mapping (I probably missed it in another review). Potted variants don't have a documented mapping as far as I know, so you can remove it from the file completely.